utils/log/loggers/fields.go

```diff
@@ -40,7 +40,21 @@ func AddToLogContext(ctx context.Context, key string, value interface{}) context
 	return ctx
 }
 
-//FromContext fetchs log fields from provided context
+//RemoveFromLogContext removes log fields from the context.
+func RemoveFromLogContext(ctx context.Context, key string) context.Context {
+	data := FromContext(ctx)
+	if data == nil {
+		ctx = context.WithValue(ctx, contextKey, make(LogFields))
```

Comment:
why create the context if it doesnt exist already?
whats the use case for the whole function?